elements/pvpindicator.lua

```diff
@@ -71,7 +71,7 @@ local function Update(self, event, unit)
 
 		status = 'ffa'
 	elseif(factionGroup and factionGroup ~= 'Neutral' and UnitIsPVP(unit)) then
-		if(UnitIsMercenary(unit)) then
+		if(unit == 'player' and UnitIsMercenary(unit)) then
```

Comment:
For more info, see [here](https://github.com/Gethe/wow-ui-source/blob/live/FrameXML/PlayerFrame.lua#L150-L157), this swap is unique to player frame.